converter/src/main/java/gov/cms/qpp/conversion/validate/CpcPerformancePeriodValidation.java

```diff
@@ -13,6 +14,8 @@
  */
 @Validator(value = TemplateId.REPORTING_PARAMETERS_ACT, program = Program.CPC)
 public class CpcPerformancePeriodValidation extends NodeValidator {
+	private static String REPORTING_PERIOD_START = Context.REPORTING_YEAR + "0101";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "REPORTING_PERIOD_START" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)